### PR TITLE
Add cessation reason configuration.

### PR DESCRIPTION
**Pull request by eschlange**
* Added new controller.
* Added specs for controller.
* Added page.
* Added configuration for all new files.

[#95521996]

### CODE REVIEW

File: src/js/controllers/CessationReasonsController.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+(function() {
+    'use strict';
+
+    function CessationReasonsController() {
+        this.CESSATION_REASONS_KEY = 'cessation_reasons';
+
+        this.cessationReason = {};
+        this.cessationReasons = localStorage.getItem(this.CESSATION_REASONS_KEY) ? JSON.parse(localStorage.getItem(this.CESSATION_REASONS_KEY)) : [];
```
Comment (ericcf) [inline src/js/controllers/CessationReasonsController.js]: Is there a reason for eagerly loading `cessationReasons` from local storage as opposed to lazily?

Comment (eschlange) [inline src/js/controllers/CessationReasonsController.js]: I'm displaying the reasons wherever this controller is used, but I can see your point. Should I simply create an accessor which does this work?

Comment (ericcf) [inline src/js/controllers/CessationReasonsController.js]: That would be my approach because `cessationReasons` is mutable.

Comment (eschlange) [inline src/js/controllers/CessationReasonsController.js]: OK, making the update now.

**Comment by eschlange** [comment]
@ericcf I made the controller more functional and changed it to lazily load cessation reasons.

**Comment by ericcf** [comment]
:ice_cream:

**Pull request merged**